### DISCUSSION TITLE
How scalable is Flet?

### DISCUSSION THREAD

**Discussion by metaperl** [flet-dev/flet#700] [Q&A]
A [user testing the waters with Flet](https://www.reddit.com/r/Python/comments/zhea3t/has_anyone_used_flet_how_robust_is_it_and_how/) expressed the following concern:

> It seems more flexible and extensible *(than Streamlit)* but I'm not sure how scalable it is because from what I can see you deploy it as a single instance so if there are lots of requests you might run into trouble (might be wrong).

**Comment by kkinder** [comment]
I don't imagine anything could stop you from deploying it to multiple instances?
